docs/conf.py

```diff
@@ -139,7 +158,7 @@
 # (source start file, name, description, authors, manual section).
 man_pages = [
     (master_doc, 'minos',
-     'Minos Microservice SAGA Documentation',
+     'Minos Microservice Common Documentation',
```

Comment:
Shouldn't this be `Saga`?
```suggestion
     'Minos Microservice Saga Documentation',
```